scripts/download-number-literals.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# integer literals
+curl https://raw.githubusercontent.com/WebAssembly/spec/master/test/core/int_literals.wast \
+	| head -n70 \
+	| grep -oP "(i(32|64).const\K\s*((\d|-|x|_|\+)*))" \
+	> packages/wast-parser/test/tokenizer/raw/int_literals.txt
+
```

Comment:
We could fetch it from the submodules in this repo.

Reply:
I'm not familiar with Git submodules, how would you do that?

Reply:
We can leave as is for now, because there's #156 and it's better to have that decoupled.